src/main/java/uk/gov/hmcts/darts/arm/client/ArmRpoClient.java

```diff
@@ -115,4 +118,11 @@ RemoveProductionResponse removeProduction(@RequestHeader(AUTHORIZATION) String b
     )
     ExtendedProductionsByMatterResponse getExtendedProductionsByMatter(@RequestHeader(AUTHORIZATION) String bearerToken,
                                                                        @RequestBody String body);
+
+    @GetMapping(value = "${darts.storage.arm-api.rpo-url.download-production-path}")
+    @SuppressWarnings({"PMD.UseObjectForClearerAPI"})
+    feign.Response downloadProduction(@RequestHeader(AUTHORIZATION) String bearerAuth,
```

Comment:
From [this](https://github.com/OpenFeign/feign/issues/1243) thread it looks like the `feign.Response` loads the body into memory rather than buffering it as a stream. From what I can it looks like a limitation of Feign. Probably something we want to keep in mind when it comes to QAing this.